frontend/ui/widget/inputdialog.lua

```diff
@@ -391,6 +391,7 @@ function InputDialog:init()
         scroll_callback = self._buttons_scroll_callback, -- nil if no Nav or Scroll buttons
         scroll = true,
         scroll_by_pan = self.scroll_by_pan,
+        has_nav_bar = self.add_nav_bar,
```

Comment:
Not very important, but to me it's either navbar or navigation bar.  ;-)

Reply:
Yeah, that's an snafu in the original code (`add_nav_bar`).

Reply:
Bah, it's just an abbreviation of add_navigation_bar :) Would you have prefered add_n8n_bar ? :)
("navbar" as a standalone word means nothing to me - it's like if we were writing "menuitem" instead of "menu_item")